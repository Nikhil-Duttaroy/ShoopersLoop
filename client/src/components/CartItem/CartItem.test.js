import React from "react";
import { shallow } from "enzyme";
import { CartItem } from './CartItem.component';

it("should render CartItem component", () => {
  const mockItem = {
    imageUrl: "www.test.com",
    price: 100,
    name: "jeans",
    quantity: 1,
  };

  expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
});
