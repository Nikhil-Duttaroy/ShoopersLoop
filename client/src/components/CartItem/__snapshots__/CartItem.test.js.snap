// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CartItem component 1`] = `
<styled.div>
  <styled.img
    alt="item"
    src="www.test.com"
  />
  <styled.div>
    <span>
      jeans
    </span>
    <span>
      1
       x $
      100
    </span>
  </styled.div>
</styled.div>
`;
